src/Configurator.php

```diff
@@ -169,6 +174,19 @@ public function withSetting($name, $value)
     }
 
     /**
+     * @param $extension string
+     * @param $className string
+     *
+     * @return Configurator
```

Comment:
From https://phpdoc.org/docs/latest/guides/types.html
$this
    This exact object instance, usually used to denote a fluent interface.